hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1522,6 +1522,19 @@
     </description>
 </property>
 
+<property>
+  <name>fs.s3a.metadatastore.fail.on.write.error</name>
```

Comment:
We should use authoritative mode instead of adding a new protperty for this.
There's no case when you want this off when auth mode is on, and on if auth mode is off.